### ISSUE TITLE
Physical/digital copy indicator

### ISSUE THREAD

**Issue by flcouselo** [a-busman/Backlogger#45]
Hi, the app is looking good. Keep up the good work. I would love to have an option to mark a game as a physical or digital copy, an maybe have an indicator (like a small icon) in the list view. I hope you take this into consideration. Thanks!

**Comment by a-busman** [comment]
That would be good. You can already choose the platform differently between digital and physical (eg. Wii U vs. Wii U eShop), but combining those would probably be better, with a p/d indicator.

**Comment by flcouselo** [comment]
Yes, that's what I've been doing but I believe that this option would be "cleaner". Thanks for your feedback.
